### ISSUE TITLE
Troops Count 

### ISSUE THREAD

**Issue by Mondster** [norecha/pokubot#75]
Looks like the troops count indeed have a bug and unable to count and deploy troops if no king is at the loot. 

[07.04.15 10:21:25 AM] INFO: StateAttack 
[07.04.15 10:21:27 AM] INFO: [gold: 269602, elixir: 148246, de: 255] 
[07.04.15 10:21:31 AM] INFO: [Troop count: [0, 89, 0, 0]] 
[07.04.15 10:21:36 AM] INFO: StateAttack 
[07.04.15 10:21:38 AM] INFO: [gold: 93651, elixir: 83643, de: 739] 
[07.04.15 10:21:42 AM] INFO: [Troop count: [0, 89, 0, 0]] 
[07.04.15 10:21:45 AM] INFO: StateAttack 
[07.04.15 10:21:48 AM] INFO: [gold: 45142, elixir: 69836, de: 487] 
[07.04.15 10:21:52 AM] INFO: [Troop count: [0, 89, 0, 0]] 
[07.04.15 10:21:56 AM] INFO: StateAttack 
[07.04.15 10:21:58 AM] INFO: [gold: 36339, elixir: 45919, de: 609] 
[07.04.15 10:22:02 AM] INFO: [Troop count: [0, 89, 0, 0]] 
[07.04.15 10:22:06 AM] INFO: StateAttack 
[07.04.15 10:22:08 AM] INFO: [gold: 205791, elixir: 134196, de: 777] 
[07.04.15 10:22:12 AM] INFO: [Troop count: [0, 89, 0, 0]] 

![04-jul-15 10-26-31 am](https://cloud.githubusercontent.com/assets/13143891/8506138/ce32a60e-2237-11e5-811d-4d00c1921556.png)
![04-jul-15 10-26-50 am](https://cloud.githubusercontent.com/assets/13143891/8506137/ce317f7c-2237-11e5-9588-29583fbc8646.png)

**Comment by norecha** [comment]
I'm sure it is not about king but about the number. It probably cannot parse 111 correctly.
Try with 111, 89 and then try with a different barb number and let me know if that is the case.

**Comment by norecha** [comment]
And while this happens, your barbs are selected right?

**Comment by Mondster** [comment]
i only have 2 barracks working for now because 2 is under upgrading. how do i try with other number for bar? i thought the troops is automatically created.

when this happen barbs are not select it just skip to select the archs to loot.

**Comment by Mondster** [comment]
![04-jul-15 10-44-39 am](https://cloud.githubusercontent.com/assets/13143891/8506164/c7cb5638-2239-11e5-85bf-1ed11234df41.png)

**Comment by norecha** [comment]
By selected I mean little white box around the unit. Selected group has a bigger portrait.

You just need to have barb other than 111. Just go to your army camp, delete a couple of them

**Comment by Mondster** [comment]
you are right looks like 111 unable to pass. i try it with 110 and is work. 

![04-jul-15 11-45-50 am](https://cloud.githubusercontent.com/assets/13143891/8506357/65200c6e-2242-11e5-9791-28c9fa3741fa.png)
![04-jul-15 11-44-56 am](https://cloud.githubusercontent.com/assets/13143891/8506356/651eeee2-2242-11e5-9c6e-575a4cec2da5.png)

**Comment by norecha** [comment]
I made a fix for it, can you test it if I send you fixed version?

**Comment by Mondster** [comment]
Sure.

**Comment by norecha** [comment]
whats your email

**Comment by Mondster** [comment]
raymond@sgoceanus.com
